apps/dashboard/src/app/(dashboard)/dashboard/connect/ecosystem/[ecosystemId]/components/client/add-partner-form.client.tsx

```diff
@@ -0,0 +1,126 @@
+"use client";
+import { Button } from "@/components/ui/button";
+import {
+  Form,
+  FormControl,
+  FormDescription,
+  FormField,
+  FormItem,
+  FormMessage,
+} from "@/components/ui/form";
+import { Input } from "@/components/ui/input";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm } from "react-hook-form";
+import { toast } from "sonner";
+import { z } from "zod";
+import { useAddPartner } from "../../hooks/use-add-partner";
+
+const formSchema = z.object({
+  name: z.string().min(1, {
+    message: "Partner name is required",
+  }),
+  domains: z.string().optional(),
+  bundleIds: z.string().optional(),
```

Comment:
can we validate these better? should be relatively easy to specify that we want a list of strings, comma separated etc?